src/frontend/state/AchievementState.ts

```diff
@@ -77,7 +77,6 @@ class AchievementState {
       {},
       {
         deep: true,
-        proxy: false,
```

Comment:
why was this removed?

Reply:
It says that the property doesn't exists so gives eslint error.
Maybe was removed on the latest mobx version.

Reply:
so I added this back and then removed it again after reading the docs:

> proxy: false forces observable(thing) to use non-[proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) implementation. This is a good option if the shape of the object will not change over time, as non-proxied objects are easier to debug and faster. **This option is not available for make(Auto)Observable**, see [avoiding proxies](https://mobx.js.org/observable-state.html#avoid-proxies).

So Typescript was not wrong in the end and this method has no `proxy` option, only the `observable` does.